core/editor/loader/v2/editor-v2-loader.php

```diff
@@ -0,0 +1,195 @@
+<?php
+namespace Elementor\Core\Editor\Loader\V2;
+
+use Elementor\Core\Editor\Loader\Common\Editor_Common_Scripts_Settings;
+use Elementor\Core\Editor\Loader\Editor_Base_Loader;
+use Elementor\Core\Utils\Assets_Config_Provider;
+use Elementor\Core\Utils\Collection;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Editor_V2_Loader extends Editor_Base_Loader {
+	const APP = 'editor';
```

Comment:
```suggestion
	const EDITOR = 'editor';
```
maybe?

Reply:
I prefer the word App, but lets talk about it